change-notes/1.20/analysis-python.md

```diff
@@ -0,0 +1,29 @@
+# Improvements to Python analysis
+
+
+ ## General improvements
+
+ > Changes that affect alerts in many files or from many queries
+> For example, changes to file classification
+ ## New queries
+
+ | **Query**                   | **Tags**  | **Purpose**                                                        |
+|-----------------------------|-----------|--------------------------------------------------------------------|
+| Default version of SSL/TLS may be insecure (`py/insecure-default-protocol`) | security, external/cwe/cwe-327 | Results are shown on LGTM by default. |
+| Use of insecure SSL/TLS version (`py/insecure-protocol`) | security, external/cwe/cwe-327 | Results are shown on LGTM by default. |
```

Comment:
Please add a very brief description of the purpose of each of these queries to the **Purpose** column.

Reply:
Ah, I completely forgot. Will do.